source/jormungandr/jormungandr/utils.py

```diff
@@ -55,6 +55,7 @@
 PY3 = sys.version_info[0] == 3
 
 DATETIME_FORMAT = "%Y%m%dT%H%M%S"
+UTC_DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%S%z"
```

Comment:
```suggestion
UTC_DATETIME_FORMAT = "%Y%m%dT%H%M%SZ"
```
I agree that local date times have to be expressed in UTC. 
But, the **format** needs to be consistent across the API. As per our documentation : 
```
Navitia exposes every date times as local times of the coverage via an ISO 8601 “YYYYMMDDThhmmss” string
```
Hence, `2012-06-15T11:00:00+0000` (the extended format) can be expressed using the base format by appending a `Z` at the end as :  `20120615T110000Z`.

https://en.wikipedia.org/wiki/ISO_8601#Coordinated_Universal_Time_(UTC)

Reply:
AFter discussion with @kadhikari , we will leave it that way, and come back to it with a proper way of generating deeplinks based on the connector.